static/app/components/u2f/u2finterface.tsx

```diff
@@ -39,7 +41,7 @@ class U2fInterface extends React.Component<Props, State> {
   };
 
   async componentDidMount() {
-    const supported = await u2f.isSupported();
+    const supported = !!window.PublicKeyCredential;
```

Comment:
is this backwards compatible with how u2f support works?

Reply:
ah this is independent of u2f-api. i need to revert it to how it was if the FF isn't enabled

Reply:
yeah overall, would double check flow still works as expected with webauthn disabled and u2f only. could see if you could configure your browser to emulate one in that situation